lms/static/sass/views/_program-marketing-page.scss

```diff
@@ -1,1767 +1,179 @@
-        font-weight: bold;
-      }
+    .section{
+      margin-bottom:20px;
     }
 
-    // Program catalog listing
-
-    .program-list {
-      .programs-listing-item {
-        box-sizing: border-box;
-        box-shadow: 1px 2px 5px #ccc;
-        position: relative;
-        height: 360px;
-        overflow: visible;
-        min-height: 0;
-        border: none;
-        display: block;
-        margin: 0 auto 40px;
-        background: $white;
-        border-radius: 0;
-
-        .program-image {
-          height: 142px;
-          position: relative;
-          overflow: hidden;
-
-          .cover-image {
-            height: 142px;
+    .facts{
+      border: 1px solid theme-color('dark');
 
-            img {
-              width: 100%;
-              height: auto;
-            }
-
-            &:before {
-              content: '';
-              position: absolute;
-              left: 0;
-              right: 0;
-              top: 0;
-              bottom: 0;
-              margin: auto;
-              background: black;
-              opacity: 0;
-              transition: all 0.2s ease-out;
-            }
-            .learn-more {
-              left: calc(50% - 100px);
-              box-sizing: border-box;
-              position: absolute;
-              z-index: 100;
-              top: 62px;
-              padding: 0 20px;
-              width: 200px;
-              height: 50px;
-              border-color: #0074b4;
-              border-radius: 3px;
-              background: #0074b4;
-              color: #fff;
-              line-height: 50px;
-              text-align: center;
-              opacity: 0;
-              text-transform: none;
-              transition: all 0.25s ease;
-            }
-          }
-        }
-
-        .banner {
-          background: #065784;
-          color: $white;
-          padding-right: 15px;
-          line-height: 18px;
-          font-weight: bold;
-          font-size: 0.7em;
-          text-align: right;
-          text-transform: uppercase;
-        }
-
-        .program-info {
-          padding: 12px 15px 5px;
-
-          .program-org {
-            font-weight: normal;
-            font-size: 0.9em;
-            color: #3d3e3f;
-            margin: 0;
-            line-height: 16px;
-          }
-
-          .program-title {
-            max-height: 55px;
-            overflow: hidden;
-            color: #222;
-            font-size: 1.25em;
-            line-height: 1.333em;
-            margin-bottom: 5px;
-          }
+      li{
+        margin: 0 20px;
+        padding: 15px 0px;
+        border-bottom: 1px solid theme-color('light');
 
-          .program-subtitle {
-            font-size: 1em;
-            margin-bottom: 33px;
-            line-height: 1.25em;
-            height: 40px;
-            color: palette(primary, dark);
-            overflow: hidden;
-          }
+        .label{
+          display:inline-block;
+          width:48%;
```

Comment:
I hate using percentage widths. :-) They always seem like a hack. Can you either use flexbox, or use the Bootstrap grid.